packages/rocketchat-livechat/client/views/app/tabbar/visitorInfo.html

```diff
@@ -32,9 +32,33 @@ <h3 title="{{username}}"><i class="status-{{status}}"></i> {{username}}</h3>
 						{{#if sms}} <li><i class="icon-mobile"></i>{{_ "SMS_Enabled"}}</li> {{/if}}
 						{{#if topic}} <li><strong>{{_ "Topic"}}</strong>: {{topic}}</li> {{/if}}
 						{{#if tags}} <li><strong>{{_ "Tags"}}</strong>: {{joinTags}}</li> {{/if}}
+						{{#if servedBy}}<li><strong>{{_ "Served by"}}</strong>: {{servedBy}}</li> {{/if}}
+						<li><strong>{{_ "ID"}}</strong>: {{_id}}</li>
```

Comment:
what is the usage for the `_id` field?

Reply:
When livechat is opened with same visitor twice, room _id can be  used to discern two rooms apart